lib/windowMaid.js

```diff
@@ -0,0 +1,86 @@
+// Third Party Modules
+const _ = require('lodash')
+
+// Logging
+const debug = require('debug')
+const logger = debug('electronpdf:')
+
+/** How long a window can remain open before it is terminated, in milliseconds */
+const HUNG_WINDOW_THRESHOLD = process.env.ELECTRONPDF_WINDOW_LIFE_THRESHOLD || 1000 * 60 * 5 /* minutes */
+
+// Window Cache - Keep track of all windows created, and if any get stuck close
+// them
+const windowCache = {}
+
+class WindowMaid {
```

Comment:
I think there are a couple benefits to using an object literal:
- It's more concise, and follows the KISS principle
- It makes the code more self-documenting by clearly indicating the intent.  Usually classes are intended to be instantiated.  If it's just a simple object, there's no chance anyone can try to instantiate it because that would be impossible.